scripts/sanity_chk/sanitylib.py

```diff
@@ -2119,18 +2317,22 @@ def process(self, message):
             self.run()
             self.instance.status, _ = self.instance.handler.get_state()
             logger.debug(f"run status: {self.instance.status}")
+
+            # to make it worth with pickle
+            self.instance.handler.thread = None
             pipeline.put({
                 "op": "report",
                 "test": self.instance,
-                "state": "executed",
```

Comment:
I can't find why this is removed and if there will be any consequences. Can you explain this change?

Reply:
it is no being used anywhere.